docs/kit/logistics/transport/procedures.md

```diff
@@ -0,0 +1,117 @@
+---
+# Procedures
+
+These are the procedures that the Kit Logistics Coordinator follows to perform their Kit Shipping responsibilities. No one else is expected to follow these procedures, they are for internal documentation only.
+
+## Shipping kits to Kickstarts
+
+To provide sufficient time for problems/delays, kits should be shipped to Kickstart locations no later than 1 week before Kickstart. Shipping the kits many weeks in advance may not be possible due to limited storage requirements at some Kickstart locations.
+
+First ensure that the Kickstart Event Coordinator is aware of the paperwork requirements \(handout forms and disclaimers\). Request the following information from the Kickstart Event Coordinator:
+
+* Name of Local Kickstart Event Coordinator
+* Email address of Local Kickstart Event Coordinator
+* Number of teams attending
+* Name of person able to receive the kit \(if not the Local Kickstart Event Coordinator\)
+* Email address of person able to receive the kit \(if not the Local Kickstart Event Coordinator\)
+* Phone number of person able to receive the kit
+* Address to ship the kit to
+
+For each Kickstart:
+
+1. Ensure that the Local Kickstart Event Coordinator, or the person that they are delegating kit management to, has received the necessary training \(to be defined\).
+2. Identify the kits to be shipped in the Inventory and move them into an appropriate `in-transit` sub directory \(e.g. `in-transit/to-oxford-ks`\).
+3. Instruct UK Postbox to forward the correct items to the recipient.
+4. Once UK Postbox have forwarded on the kits, email the LKEC and the recipient \(if not the LKEC\) with the tracking number/link and a list of kit asset numbers shipped to them. Ask them to report back when they have received the shipment and doubled checked that they have the expected kits.
+5. Once the kits have been delivered, move them in the Inventory to an appropriate sub directory \(e.g. `kickstart-venues/oxford`\).
+6. Email details of what kits are where to the Kickstart Event Coordinator so that they can prepare handout forms for each of the Local Kickstart Event Coordinators.
+
+After Kickstart, request scanned copies of the handout form and disclaimers from all of the Kickstarts from the Kickstart Event Coordinator \(who will then request them from the Local Kickstart Coordinators\). Move each kit into an appropriate sub directory in the Inventory based upon which team it was given to \(e.g. move a kit to `teams/2017/srz`\).
+
+## Shipping kits to teams that didn't attend a Kickstart
+
+The operations manual currently requires that the Kit Logistics Coordinator ensures that kits are available to be collected by a courier no later than 2 week after Kickstart.
+
+Begin by requesting the following information from the Team Coordinator for the teams that didn't attend a Kickstart and ask them to confirm that the details are correct via the appropriate LTC:
```

Comment:
What is an LTC? (I worked it out, but others might not)

Reply:
53b454f